src/synthesizer/grid.py

```diff
@@ -306,11 +306,15 @@ def _get_spectra_grid(self, spectra_to_read):
         # If a full cloudy grid is available calculate some
         # other spectra for convenience.
         if self.reprocessed:
+            # The total emission (ignoring any dust reprocessing) is just
```

Comment:
Do we actually need this? I don't know that we ever use it. Is it more of a "nice to see when plotting from the `Grid`" situation?

Reply:
I think so yeah. I'd keep it